superbench/config/azure_ndv4.yaml

```diff
@@ -2,6 +2,10 @@
 version: v0.3
```

Comment:
Shall we add to all configuration?

Reply:
I didn't add monitor related configs into AMD config file. Since the monitor doesn't support AMD.

Reply:
Got it.

Reply:
Please add a check for AMD, thanks.

Reply:
Done.